tests/test_fileio.py

```diff
@@ -3,15 +3,24 @@
 import os.path as osp
 import sys
 import tempfile
+from contextlib import contextmanager
 from unittest.mock import MagicMock, patch
 
 import pytest
 
 import mmcv
-from mmcv.fileio.file_client import HTTPBackend, PetrelBackend
+from mmcv.fileio.file_client import AWSBackend, HTTPBackend, PetrelBackend
 
-sys.modules['petrel_client'] = MagicMock()
-sys.modules['petrel_client.client'] = MagicMock()
+
+@contextmanager
+def package_mock(*package_name):
```

Comment:
It seems that this function is implemented in lots of files. Could we implement it in one file which likes `test_util.py`?

Reply:
Yes, i will fix it.